suite-common/wallet-core/src/tokenDefinitions/tokenDefinitionsActions.ts

```diff
@@ -0,0 +1,28 @@
+import { createAction } from '@reduxjs/toolkit';
+
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { TokenDefinition } from '@suite-common/wallet-types';
+
+const actionsPrefix = '@common/token-definitions';
```

Comment:
Nitpicks:
1. Based on other part of `wallet-core` it should be `@common/wallet-core/token-definitions`
2. Also the folder should be token-definitions, not tokenDefinitions

Reply:
fb056f24aeaa7c54a17ad73299fa3530e9c90036
726defcd3fdc7fc585b569e25e9b6df8d8038d3d
3f64548e333013df5180297ebed6d38377fc3902